src/Adapter/AttributeGroup/Validate/AttributeGroupValidator.php

```diff
@@ -52,7 +52,9 @@ public function __construct(
      */
     public function validate(AttributeGroup $attributeGroup): void
     {
+        $this->validateObjectModelLocalizedProperty($attributeGroup, 'name', AttributeGroupConstraintException::class, AttributeGroupConstraintException::EMPTY_NAME);
```

Comment:
You can't really do this, the validate method returns a boolean but you don't know for which reason so by adding this constraint associated to "empty name" error you remove the one associated to "invalid name"

To handle both type of invalidate I think you need to check manually the empty constraint and throw the exception manually, then you can use the `validateObjectModelLocalizedProperty` for the other invalidaty